src/components/AccountLoginForm.jsx

```diff
@@ -68,6 +70,12 @@ const AccountLoginForm = ({ t, isOAuth, oAuthTerminated, fields, error, dirty, s
           }
         }
       )}
+      { values.password && !hasGlobalError &&
```

Comment:
We do not rely on `password` to know if an account is already configured. For example, an OAuth konnector does not have a `password`. Instead, we use the `editing` property in `AccountConnection`, which already contains all the cases.